CordovaLib/Classes/Private/Plugins/CDVIntentAndNavigationFilter/CDVIntentAndNavigationFilter.m

```diff
@@ -95,13 +95,8 @@ - (CDVIntentAndNavigationFilterValue) filterUrl:(NSURL*)url
     return [[self class] filterUrl:url intentsWhitelist:self.allowIntentsWhitelist navigationsWhitelist:self.allowNavigationsWhitelist];
 }
 
-#if WK_WEB_VIEW_ONLY
 #define CDVWebViewNavigationTypeLinkClicked 0
 #define CDVWebViewNavigationTypeOther 5
```

Comment:
Do these get used by our WKWebView implementation?

Reply:
```
#define CDVWebViewNavigationTypeOther 5
```
Is a UIWebView specific value. For WKWebView, the WKNavigationType Other value is `-1` and not `5`. I have updated this value to represent the correct value for WKWebView.

The result of this value is used in the `shouldOverrideLoadWithRequest` method.

It is what determines if a link should load when coming from something other than an anchor tag. For example using `location.href` in JS. It is apart of the `allow-intent` stuff.